doc/guide/build_windows.hpp

```diff
@@ -110,7 +132,7 @@ compiler version, check if the Visual Studio compiler and Windows SDK are instal
 - Build > Build Solution
 - Once it has successfully finished, close Visual Studio
 
-@section build_windows_mlpack Building mlpack
+@section build_windows_mlpack Building mlpack with CMake's Generated Solution
```

Comment:
```suggestion
@section build_windows_mlpack Building mlpack with a CMake-Generated Solution
```
This is *really pedantic* (sorry!).  Up to you if you want to take the suggestion.

Reply:
Thanks, I have rewritten the above. Don't worry, you can never have too many improvement suggestions, each discussion improves the quality of the code/documentation!